controller/index.js

```diff
@@ -1,1019 +1,1054 @@
+      _sendImmediate(file)
     }
+  }
 
-    // build the geometry from z,x,y
-    var bounds = merc.bbox( req.params.x, req.params.y, req.params.z );
-    req.query.geometry = {
-        xmin: bounds[0],
-        ymin: bounds[1],
-        xmax: bounds[2],
-        ymax: bounds[3],
-        spatialReference: { wkid: 4326 }
-    };
-
-    // Get the tile and send the response to the client
-    var _send = function( err, info ){
-      req.params.key = key;
-      req.params.type = 'agol';
-      agol.getServiceTile( req.params, info, function(err, tile){
-        if ( err ){
-          res.status(401).send( err );
-          return;
-        }
-        res.sendfile( tile );
-      });
-    };
-
-    // file key tells is a combo key for standardizing look ups in the fs.system 
-    var key = [req.params.item,'all'].join(':');
-    // build the names of the files 
-    // Note that the layer id would be present in service level tiles 
-    var file = agol.files.localDir + '/tiles/';
-      file += key + '/' + req.params.format;
-      file += '/' + req.params.z + '/' + req.params.x + '/' + req.params.y + '.' + req.params.format;
-
-    // if the json file alreadty exists, dont hit the db, just send the data
-    if ( fs.existsSync( file ) ){
-      res.sendfile( file );
-    } else  {
-      agol.find( req.params.id, function( err, data ){
-        if (err) {
-          res.status(500).send( err );
-        } else {
-
-          // sort the req.query before we hash so we are consistent 
-          var sorted_query = {};
-          _(req.query).keys().sort().each(function (key) {
-            if (key != 'url_only' && key != 'format'){
-              sorted_query[key] = req.query[key];
-            }
-          });
-          // Get the item
-          agol.getServiceLayerData( data.host, req.params.item, null, req.query, function(error, itemJson){
-            if (error) {
-              res.status(500).send( error );
-            } else {
-              // send the data for the tile to Tiles 
-              _send(error, itemJson);
-            }
-          });
-        }
-      });
-    }
-
-  };
-
-  // drops the cache for an item and DELETEs all known files
-  controller.deleteItemData = function(req, res){
-    req.query.forceDelete = true;
-    controller.dropItem(req, res);
-  };
-
-  controller.getQueueCounts = function(req, res){
-    agol.getQueueCounts(function(err, response){
-      if (err){
-        res.status(500).send(err);
-      } else {
-        res.json(response);
-      }
-    });
-  };
+  /**
+   * Forced a drop of the cache for an item and DELETEs all known files
+   * used for responding to DELETE calls.
+   */
+  controller.deleteItemData = function (req, res) {
+    req.query.forceDelete = true
+    controller.dropItem(req, res)
+  }
 
-   /** 
-   * Get the geohash for an item/layer
+  /**
+   * Request handler for returning the counts of currently queued workers
    *
    */
-  controller.getGeohash = function(req, res){
```

Comment:
where did this silent param come from? I've seen it a few times and I'm not sure what its deal is